manifests/blacksmith/shield-backups.yml

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  shield:
+    exodus: (( concat $GENESIS_EXODUS_MOUNT params.shield.deployment_env "/" params.shield.deployment_type ":" ))
```

Comment:
In order to allow the user to override the exodus endpoint, you should provide this feature via params.  One way of doing this would be setting `meta.shield.default_exodus` to this value, then have `params.shield_exodus_path` set to `(( grab meta.shield.exodus ))`, then using `params.shield_exodus_path` used in the vault commands below.